app/models/invoice.rb

```diff
@@ -4,5 +4,5 @@ class Invoice < ApplicationRecord
   has_many :invoice_items
   has_many :items, through: :invoice_items
 
-  enum status: [ :in_progress, :completed, :cancelled ]
+  enum status: ["in progress", "completed", "cancelled"]
```

Comment:
I'm wondering if we should change all enums to this format or keep the others as symbols